src/trufflecon2020.html

```diff
@@ -102,16 +114,16 @@ <h2>Grow with the Blockchain Community</h2>
 
 </main>
 
-<main class="banner banner-tcon-2020 pt-5 pb-5">
+<main class="banner-tcon-2020 pt-5 pb-5">
```

Comment:
I believe https://github.com/trufflesuite/trufflesuite.com/pull/786/files#diff-3f9521b51cadf0a09991e141b6b597efR186 makes this to be the following change, but I'm assuming you tested this? If it works without the `-truffle`, then I'd argue what the point is for the class in general

```suggestion
<main class="banner-truffle-tcon-2020 pt-5 pb-5">
```